frontend/package.json

```diff
@@ -36,7 +38,8 @@
     "semantic-ui-react": "^2.0.1",
     "styled-components": "^5.2.1",
     "suneditor-react": "^2.16.0",
-    "web-vitals": "^1.1.1"
+    "web-vitals": "^1.1.1",
+    "yup": "^0.32.9"
```

Comment:
please remove all the unnecessary packages(including this)

Reply:
Yup is a necessary Library to use for validation , this is the only thing I have used with React-Hook-Form  for Join Us Form Validation